Project/verify_dashboard.py

```
@@ -0,0 +1,146 @@
+            print(f"Error: Dashboard server not found at http://localhost:8000. Please start it first.")
+            print(f"  Command: PYTHONPATH=Project python -m uvicorn main:app --app-dir Project --port 8000")
+            exit(1)
+        finally:
+            conn.close()
+
+        browser = p.chromium.launch(headless=True)
+        context = browser.new_context(viewport={'width': 1280, 'height': 800})
+        page = context.new_page()
+        try:
+            test_all_features(page)
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)